Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -210,6 +211,16 @@ namespace RedirectOutput
         });
 
         SetRedirection("stderr", g_redirect_stderr_saved, g_redirect_stderr, []([[maybe_unused]] const char* msg) {
+            if (lastPythonError.empty())
+            {
+                lastPythonError = msg;
+                const int lengthOfErrorPrefix = 11;
+                auto errorPrefix = lastPythonError.find("ERROR:root:");
+                if (errorPrefix != AZStd::string::npos)
+                {
+                    lastPythonError.erase(errorPrefix, lengthOfErrorPrefix);
+                }
+            }
```

Comment:
Here, I recommend appending to the list/vector of errors after removing the substring "ERROR:root:", which might be a general pattern in logs that we don't want?